src/Microsoft.Health.Dicom.Tests.Common/Extensions/DicomTagExtensions.cs

```diff
@@ -25,5 +28,17 @@ public static ExtendedQueryTagStoreEntry BuildExtendedQueryTagStoreEntry(this Di
         {
             return new ExtendedQueryTagStoreEntry(key: key, path: tag.GetPath(), vr: vr ?? tag.GetDefaultVR().Code, privateCreator: privateCreator, level: level, status: status, queryStatus: QueryStatus.Enabled, errorCount: 0);
         }
+
+        public static WorkitemQueryTagStoreEntry BuildWorkitemQueryTagStoreEntry(string path, int key, string vr)
+        {
+            var dicomTagParser = new DicomTagParser();
+            var entry = new WorkitemQueryTagStoreEntry(key, path, vr);
+            if (dicomTagParser.TryParse(path, out var dicomTags))
+            {
+                entry.PathTags = Array.AsReadOnly(dicomTags);
```

Comment:
Is this necessary?

Reply:
Yes it is necessary to populate the pathTags, since it is a readOnly collection, this is the way we found to populate